safe-global/safe-wallet-web
# Tests: Address book tests rem p1

## What it solves

Resolves #3229 

## How this PR fixes it

- Added tests to cover csv import in Address Book

Tests added:

- Verify that importing a csv file does not alter addresses in the Address book not present in the file
- Verify Safe name changes after uploading a csv file
- Import a csv file with an empty address/name/network in one row
- Import a non-csv file
- Import a csv file with a repeated address and same network
- Verify modal shows the amount of entries and networks detected

## How to test it

- Run Cypress tests and observe outcome
